requirements.txt

```diff
@@ -1 +1,6 @@
 Flask==1.1.1
+flask-restful
```

Comment:
Can you please pin the versions for all python modules?

Reply:
done